tests/python_package_test/test_sklearn.py

```diff
@@ -74,10 +76,35 @@ def multi_logloss(y_true, y_pred):
     return np.mean([-math.log(y_pred[i][y]) for i, y in enumerate(y_true)])
 
 
+@lru_cache(maxsize=None)
+def _load_boston(**kwargs):
+    return load_boston(**kwargs)
+
+
+@lru_cache(maxsize=None)
+def _load_breast_cancer(**kwargs):
+    return load_breast_cancer(**kwargs)
+
+
+@lru_cache(maxsize=None)
+def _load_digits(**kwargs):
+    return load_digits(**kwargs)
+
+
+@lru_cache(maxsize=None)
+def _load_iris(**kwargs):
+    return load_iris(**kwargs)
+
+
+@lru_cache(maxsize=None)
+def _load_linnerud(**kwargs):
+    return load_linnerud(**kwargs)
```

Comment:
Can we move all these proxies to `utils.py` with the aim to deduplicate code?

Reply:
oh sure, that's a good point!